cmd/flipt/config.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"os"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/spf13/cobra"
+	"go.flipt.io/flipt/internal/config"
+	"gopkg.in/yaml.v2"
+)
+
+func newConfigCommand() *cobra.Command {
+	var configCmd = &cobra.Command{
+		Use:   "config",
+		Short: "Manage Flipt configuration",
+	}
+
+	var initCmd = &cobra.Command{
```

Comment:
i made this a subcommand because i'd also like to add others in the future like `flipt config validate`